doc/source/make_tables.py

```diff
@@ -155,12 +155,14 @@ def generate(cls):
     @classmethod
     def fetch_data(cls):
         """Get a list of row_data used to generate the table."""
-        raise NotImplementedError('Subclasses should specify a fetch_data method.')
+        msg = 'Subclasses should specify a fetch_data method.'
+        raise NotImplementedError(msg)
```

Comment:
What's the justification behind all of these changes? This feels wrong to me and unnecessarily changing 1000+ lines just to break up the message and raising the exception?

Reply:
This PR will make errors easier to understand and trace back. 
See https://docs.astral.sh/ruff/rules/f-string-in-exception/